lib/buckets/common.js

```diff
@@ -19,7 +19,7 @@ var format = util.format;
 var errors = require('../errors');
 
 var LIST_LIMIT = 1024;
-var BORAY_MULTIPLIER = 2;
+var BUCKETS_MDAPI_MULTIPLIER = 2;
 
 /*
  * A valid bucket name is composed of one or more "labels," separated by
```

Comment:
nit: there is a "boray" ref in a comment on line 70 here.

Reply:
Fixed